analog-brawnish/usr/share/asteroid-launcher/watchfaces/analog-brawnish.qml

```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (C) 2022 - David Rinehart wdaver(at)cox.net
```

Comment:
The RotM and CenterX/Y variables later on sound rather familiar. If you built this watchface using an example with given credits, it would be nice to the previous contributors to keep those mentions according to the license and extend your credits on top.
Don't take this as an offense please, if you just used the variable names but consider the code your sole work, there is nothing to complain ;)